controller/controller_test.go

```diff
@@ -245,22 +248,23 @@ var _ = Describe("Controller", func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(instanceIP).ToNot(BeNil())
 				Expect(instanceIP.GetUuid()).To(Equal(testInstanceIP.GetUuid()))
+				Expect(instanceIP.GetInstanceIpAddress()).ToNot(Equal(""))
```

Comment:
Why are you checking not empty string? Maybe Expect(instanceIP.GetInstanceIpAddress()).To(Equal(testInstanceIP.GetInstanceIpAddress())) ?